socorro/external/es/base.py

```diff
@@ -4,12 +4,10 @@
 
 import datetime
 
-from configman import Namespace, class_converter
+from configman import Namespace, class_converter, RequiredConfig
 
-from socorro.app.socorro_app import App
 
-
-class ElasticsearchConfig(App):
+class ElasticsearchConfig(RequiredConfig):
```

Comment:
Yeah... so this used to subclass `App`. The problem with that is if you don't mock out the Elasticsearch implementation for the crashstats models, then it tries to configure this and this picks up the configuration from `App` which includes a logger set up to talk to the syslog service and there isn't one when the tests are running and the resulting traceback reads like a chapter from a Chuck Palhuniak book that was deleted because it was just too cray-cray.

I don't think anything should be subclassing `App` unless it's an app, so I switched it to subclass `RequiredConfig` instead. That made some of my problems go away.